supervisor-services-labs/argocd-operator/argocd-instance.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: my-demo-argocd
+  namespace: ${{MY_VSPHERE_NAMESPACE}}
+  labels:
+    example: basic
+spec:
+  nodePlacement:
+    nodeSelector:
+      kubernetes.io/os: CRX
+  server:
+    service:
+      type: LoadBalancer
+# Currently redis images are downloaded from Dockerhub and may be subject to download limits. To overcome this, use a proxy cache as an example
+  redis:
+    image: ${{MY_PROXY_CACHE_REPOSITORY}}/proxy-cache/library/redis
```

Comment:
Can we use docker hub as default?
It sounds like this is a templated file but with no default values?

Reply:
I could remove the section altogether. I will add this section as a side note if the user hits the Docker rate limit.